conf/dhcp_filters.conf.example

```diff
@@ -1,47 +1,292 @@
-# Dhcp filter configuration
-# 
-# you are able to take an action based on the dhcp information. 
-#
-# Make a simple rule like this:
-#
-#[VIVIDE9]
-#filter = computername
-#operator = regex
-#value = ^VIVIDE9
-#
-#[mac]
-#filter = mac
-#operator = regex
-#value = ^00:0b:ab
-#
-#[dhcp_fingerprint]
-#filter = dhcp_fingerprint
-#operator = is
-#value = 1,15,3,6,44,46,47,31,33,249,43
+# Dhcp FILTERS configuration
+# ==========================
 #
-#[dhcp_vendor]
-#filter = dhcp_vendor
-#operator = is
-#value = MSFT 5.0
+# Scopes
+# ------
 #
-#This rule will change the device type and device class for this specific MAC address
+# It's possible to return custom attributes in the dhcp answer in the specific scopes
+# - Discover: Rewrite the dhcp Offer
+# - Request: Rewrite the dhcp ACK
 #
-#[1:VIVIDE9&mac&dhcp_fingerprint&dhcp_vendor]
-#scope = Fingerbank
-#action = modify_node
-#action_param = mac = $mac,  device_type = GE Vivid e9,  device_class = Medical
+# A "Dhcp filter" configuration is mainly constituted of two (2) parts; rules(s)/condition(s) and action(s)
+# Some working examples covering useful use cases can be found at the bottom of this file
+# 
+# Rule(s)/Condition(s)
+# --------------------
+# Structure example of a rule/condition
+# [NAMEOFRULE]
+# filter = FILTER
+# operator = OPERATOR
+# value = VALUE
 #
-# filter can be:
+# Filter can be:
+#     options.optionVendorClassIdentifier
```

Comment:
Could you get examples and possible values for the options

Reply:
There is an example below

Reply:
#          'options' => {
#                         'optionVendorClassIdentifier' => 'android-dhcp-8.1.0',
#                         'optionDHCPMessageType' => '1',
#                         'optionClientIdentifier' => '0178f8829f119d',
#                         'optionParameterRequestList' => '1,3,6,15,26,28,51,58,59,43',
#                         'optionMaximumDHCPMessageSize' => '1500'
#                       },